src/MongoDB.Driver.Core/Core/Misc/Feature.cs

```diff
@@ -42,6 +42,7 @@ public class Feature
         private static readonly Feature __aggregateOutToDifferentDatabase = new Feature("AggregateOutToDifferentDatabase", new SemanticVersion(4, 3, 0));
         private static readonly Feature __aggregateToString = new Feature("AggregateToString", new SemanticVersion(4, 0, 0));
         private static readonly Feature __aggregateUnionWith = new Feature("AggregateUnionWith", new SemanticVersion(4, 3, 4));
+        private static readonly AllowDiskUseFeature __allowDiskUse = new AllowDiskUseFeature("AllowDiskUse", new SemanticVersion(4, 4, 0, ""));
```

Comment:
Rename field and move to new position in alphabetical order:

        private static readonly FindAllowDiskUseFeature __findAllowDiskUse = new FindAllowDiskUseFeature("FindAllowDiskUse", new SemanticVersion(4, 4, 0, ""));

Reply:
Done.